lib/streamlit/commands/page_config.py

```diff
@@ -102,6 +114,31 @@ def set_page_config(
 
     msg.page_config_changed.initial_sidebar_state = initial_sidebar_state
 
+    if menu_options is not None:
+        lowercase_menu_options = {
+            k.lower().strip(): v if v else "" for k, v in menu_options.items()
+        }
+
+        menu_options_proto = msg.page_config_changed.menu_options
+        if "get help" in lowercase_menu_options:
+            if lowercase_menu_options["get help"]:
+                help_url = lowercase_menu_options["get help"]
+                help_url = fix_url(help_url)
+                menu_options_proto.get_help_url = help_url
+            else:
+                menu_options_proto.hide_get_help = True
+
+        if "report a bug" in lowercase_menu_options:
+            if lowercase_menu_options["report a bug"]:
+                bug_url = lowercase_menu_options["report a bug"]
+                bug_url = fix_url(bug_url)
+                menu_options_proto.report_a_bug_url = bug_url
+            else:
+                menu_options_proto.hide_report_a_bug = True
+
+        if "about" in lowercase_menu_options:
+            menu_options_proto.about_section_md = lowercase_menu_options["about"]
```

Comment:
Would recommend splitting this out into a helper function since the `set_page_config` function is getting pretty long